packages/performance/src/performance.ts

```diff
@@ -159,6 +159,16 @@ export class Performance {
           | TimeToFirstContentfulPaintEvent);
       });
     }
+
+    if (typeof perfMetrics !== 'undefined') {
+      perfMetrics.onFirstInputDelay(delay => {
+        this.lifecycleEvent({
+          type: EventType.FirstInputDelay,
+          start: 0,
```

Comment:
Can we make start `now` so we know when it was triggered? It will look weird here in the performance charts

Reply:
Or, maybe, if this is triggered after the delay end, start is `now() - delay`?